vignettes/deaths.txt

```diff
@@ -0,0 +1,36 @@
+## Definition of the time-step and output as "time"
```

Comment:
Any reason that this, and other odin models, are not a .R file? If it's just the linting drama that's tuneable, though lintr's file-ignore system seems broken to me